src/Core/src/Handlers/View/ViewHandlerOfT.cs

```diff
@@ -128,5 +129,11 @@ internal ViewHandler()
 			get => (TVirtualView?)base.VirtualView;
 			private protected set => base.VirtualView = value;
 		}
+
+		protected TVirtualView VirtualViewWithValidation([CallerMemberName] string callerName = "")
```

Comment:
This way we don't have to keep checking for null everywhere that we know the VirtualView should have a value